src/main/kotlin/net/atos/zac/task/TaskService.kt

```diff
@@ -64,6 +64,11 @@ class TaskService @Inject constructor(
                 )
                 changed = true
             }
+        } ?: run {
```

Comment:
```suggestion
        } ?: {
```

I don't think `run` or any other scope function really adds any value here?